frontend/services/users/userService.ts

```diff
@@ -0,0 +1,23 @@
+import { useMutation, UseMutationResult } from 'react-query';
+
+import { AxiosResponse } from 'axios';
+
+import { SignupDto } from 'types/signup';
+
+import USERS from '.';
+
+export function useSignup(): UseMutationResult<
+  AxiosResponse<SignupDto>,
+  unknown,
+  SignupDto,
+  unknown
+> {
+  const signup = async (dto: SignupDto): Promise<AxiosResponse<SignupDto>> => {
+    try {
```

Comment:
I don't think you should try/catch here because you won't be able to display an error in the UI.